tests/cpp/security/XmlAuthzCredentialGenerator.hpp

```diff
@@ -55,7 +55,7 @@ const opCodeList::value_type QArr[] = {OP_QUERY, OP_REGISTER_CQ};
 
 const stringList::value_type QRArr[] = {"Portfolios", "Positions"};
 
-const char* PRiUsnm = "%s%d";
+static const char* PRiUsnm = "%s%d";
```

Comment:
`constexpr` work here?

Reply:
It did not.  `ISO C++11 does not allow conversion from string literal to 'char *const'` is the specific error when trying to use it while on a mac. Haven't looked more into at the moment